updater/spec/dependabot/updater_spec.rb

```diff
@@ -10,77 +10,97 @@
 require "dependabot/service"
 
 RSpec.describe Dependabot::Updater do
-  subject(:updater) do
-    Dependabot::Updater.new(
-      service: service,
-      job_id: 1,
-      job: job,
-      dependency_files: dependency_files,
-      base_commit_sha: "sha",
-      repo_contents_path: repo_contents_path
-    )
+  # rubocop:disable Metrics/MethodLength
+  def build_job(requested_dependencies: nil, allowed_updates: default_allowed_updates,
+                existing_pull_requests: [], ignore_conditions: [], security_advisories: [],
+                experiments: {}, updating_a_pull_request: false, security_updates_only: false)
+    @build_job ||=
```

Comment:
I think this memoization doesn't make sense since the method has parameters, so multiple invocations will be ignored in favor of the first one. I'd remove it.

Reply:
Good call! That was a relic of refactoring that I became blind to 😅

Rubocop was enforcing the name, which is really awkward (IMO), so it's extra great to get rid of this 😄

Reply:
Addressed in [9f50771](https://github.com/dependabot/dependabot-core/pull/6729/commits/9f50771aaaf4e3bf8478d5b6dc199359b60c3284)